FlowCrypt/Functionality/DataManager/Local Storage/LocalStorage.swift

```diff
@@ -50,10 +50,8 @@ extension LocalStorage: LogOutHandler {
     }
 
     func cleanup() {
-        Constants.allCases
-            .map(\.rawValue)
-            .forEach {
-                storage.removeObject(forKey: $0)
-            }
+        for key in Constants.allCases.map(\.rawValue) {
+            storage.removeObject(forKey: key)
+        }
```

Comment:
I find this much clearer :+1: even if it's sometimes a bit more code. It could be due to what I'm used to. But jumping between codebases to do review, consistency really helps me understand the code faster.